.github/ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,22 @@
 ### Issue details
 
-_Please provide issue details here_.
+Please provide issue details here Remember, this is _not_ a place to ask questions. For that, go to http://gitter.im/wallabag/wallabag.
+
+### Environment
+
+* wallabag version (or git revision) that exhibits the issue:
+* Last wallabag version that did not exhibit the issue (if applicable):
+* php version:
+* OS:
+* type of hosting (shared or dedicated):
+* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): 
 
 ### Steps to reproduce/test case
 
-_Please provide necessary steps for reproduction of this issue, or better the
-reduced test case (without any external dependencies, if possible)_.
+Please provide necessary steps for reproduction of this issue, or better the
+reduced test case (without any external dependencies, if possible).
 
 ### Affected version(s)
 
-_Please fill in affected version(s)_
+Please fill in affected version(s).
```

Comment:
What is the difference with the line _wallabag version (or git revision) that exhibits the issue_ ?